### PR TITLE
fix: [CDS-52358]: modified api calls for acrartifact runtime

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger titlecheck

File: src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ACRArtifactSource/hooks/useGetSubscriptionsForAcrArtifact.ts [hunk 1]
```diff
@@ -21,71 +17,47 @@ export interface Params {
...
-    serviceId,
-    subscriptionsFqnPath
-  } = params
+  const { connectorRef, accountId, projectIdentifier, orgIdentifier } = params
 
   const {
     data: subscriptionsV1Data,
     refetch: refetchV1Subscriptions,
     loading: loadingV1Subscriptions,
     error: subscriptionsV1Error
```
Comment (PratyushGar) [inline src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ACRArtifactSource/hooks/useGetSubscriptionsForAcrArtifact.ts:37]: nit: 
```suggestion
  const {
    data: subscriptionsData,
    refetch: refetchSubscriptions,
    loading: loadingSubscriptions,
    error: subscriptionsError
```

**Review by PratyushGar — APPROVED** [review]

**Comment by sahithikolichala** [comment]
retrigger jest

File: src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ACRArtifactSource/hooks/useGetSubscriptionsForAcrArtifact.ts [hunk 2]
```diff
@@ -21,71 +17,47 @@ export interface Params {
   useArtifactV1Data?: boolean
   serviceId?: string
   subscriptionsFqnPath: string
+  pipelineRuntimeYaml: string
 }
 
 interface ReturnType {
   subscriptionsData: ResponseAzureSubscriptionsDTO | null
-  refetchSubscriptions: (options?: { queryParams: any }) => Promise<void>
+  refetchSubscriptions: any
```
Comment (pankajsingh-harness) [inline src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ACRArtifactSource/hooks/useGetSubscriptionsForAcrArtifact.ts:25]: Add proper type

**Review by pankajsingh-harness — APPROVED** [review]

**Review by satyam-harness — APPROVED** [review]

**Comment by sahithikolichala** [comment]
retrigger jest

**Pull request merged**